Source/UI/NewUI/CommonUI/View/Core/ButtonView.swift

```diff
@@ -39,6 +41,10 @@ class ButtonView: UIView {
     func update(with style: ElementButtonStyle) {
         self.style = style
         backgroundColor = style.backgroundColor
+        clipsToBounds = true
+        layer.borderColor = style.normalBorderColor.cgColor
+        layer.cornerRadius = style.cornerRadius
+        layer.borderWidth = style.borderWidth
```

Comment:
Does this now mean that UIs that use `SelectionButtonView` will now have double border, one around SelectionButtonView and another around the actual button?

Reply:
Nope. only one around the view.